manifests/certonly.pp

```diff
@@ -38,6 +38,17 @@
 # [*cron_minute*]
 #   Optional string, integer or array, minute(s) that the renewal command should execute.
 #   e.g. 0 or '00' or [0,30].  Default - seeded random minute.
+# [*pre_hook_commands*]
```

Comment:
Take care that it already exists parameters `cron_before_command` and `cron_success_command` that is functionally close but in fact totally different.

So we should take care to update `README.md` with clear examples that explains the difference.
I did not know in which version of Letsencrypt the `hook` options appeared. I think it is needed to be checked be cause we support several OSes with installation from distro packages with different versions.

Reply:
README updated, should be pretty clear and concise.